lib/tstracker/chainbs.go

```diff
@@ -137,8 +139,12 @@ func (*tcs) DeleteMany([]cid.Cid) error {
 	return errors.New("not supported")
 }
 
-func (*tcs) View(cid.Cid, func([]byte) error) error {
-	return errors.New("not supported")
+func (x *tcs) View(c cid.Cid, f func([]byte) error) error {
+	b, err := x.Get(c)
+	if err != nil {
+		return err
+	}
+	return f(b.RawData())
 }
```

Comment:
You don't need this, and the `DeleteMany` above:
https://github.com/filecoin-project/statediff/blob/122afec5e9ba0372f4241184344dd574857a7cb3/lib/tstracker/chainbs.go#L35-L36
+
https://github.com/filecoin-project/go-bs-postgres-chainnotated/blob/feat/the_force_awkens/store.go#L194-L200
```suggestion
```